vulnerabilities/importers/elixir_security.py

```diff
@@ -100,40 +92,44 @@ def get_versions_from_range(self, version_list, pkg_name):
         return pkg_versions
 
     def process_file(self, path):
-        with open(path) as f:
-            yaml_file = yaml.safe_load(f)
-            pkg_name = yaml_file["package"]
-            safe_pkg_versions = []
-            if yaml_file.get("unaffected_versions"):
-                safe_pkg_versions = self.get_versions_from_range(
-                    yaml_file["patched_versions"] +
-                    yaml_file["unaffected_versions"],
-                    pkg_name,
-                )
-            else:
-                safe_pkg_versions = self.get_versions_from_range(
-                    yaml_file["patched_versions"], pkg_name
-                )
-
-            cve_id = "CVE-"+str(yaml_file["cve"])
-            safe_purls = []
-            if safe_pkg_versions is not None:
-                safe_purls = {
-                    PackageURL(name=pkg_name, type="hex", version=version)
-                    for version in safe_pkg_versions
-                }
-
-            vuln_reference = [
-                Reference(
-                    reference_id=yaml_file["id"],
-                    url=yaml_file["link"],
-                )
-            ]
-
-            return Advisory(
-                summary=yaml_file["description"],
-                impacted_package_urls=[],
-                resolved_package_urls=safe_purls,
-                cve_id=cve_id,
-                vuln_references=vuln_reference,
+        yaml_file = load_yaml(path)
+        pkg_name = yaml_file["package"]
+        safe_pkg_versions = []
+        if yaml_file.get("unaffected_versions"):
+            safe_pkg_versions = self.get_versions_from_range(
+                yaml_file["patched_versions"] +
+                yaml_file["unaffected_versions"],
+                pkg_name,
+            )
+        else:
+            safe_pkg_versions = self.get_versions_from_range(
+                yaml_file["patched_versions"], pkg_name
+            )
+        if yaml_file.get('cve'):
+            cve_id = "CVE-" + yaml_file["cve"]
+        else:
+            cve_id = ""
+
+        safe_purls = []
+        if safe_pkg_versions is not None:
+            safe_purls = {
+                PackageURL(name=pkg_name, type="hex", version=version)
+                for version in safe_pkg_versions
+            }
+
+        vuln_reference = [
+            Reference(
+                reference_id=yaml_file["id"],
+            ),
+            Reference(
+                url=yaml_file["link"],
             )
+        ]
+
+        return Advisory(
+            summary=yaml_file["description"],
+            impacted_package_urls=[],
```

Comment:
Something is wrong here. The packages which don't belong to the set of packages which are unaffected and patched belong here

Reply:
Ok